pkg/ds/farm.go

```diff
@@ -581,8 +581,12 @@ func (dsf *Farm) spawnDaemonSet(
 
 	if dsf.monitorHealth {
 		go func() {
-			aggregateHealth := replication.AggregateHealth(ds.PodID(), *dsf.healthChecker)
-			ticks := time.NewTicker(1 * time.Minute)
+			// daemon sets are deployed to many servers and deploys are very slow. This means that
+			// 1) the data size for each health response is huge which can suck up bandwidth
+			// 2) the metrics aren't going to change very frequently, so a low sample rate is acceptable
+			watchDelay := 1 * time.Minute
```

Comment:
Yeah, we may need to increase this delay as the number of daemon sets increases.